models/sipnet/R/read_restart.SIPNET.R

```diff
@@ -45,17 +45,17 @@ read_restart.SIPNET <- function(outdir, runid, stop.time, settings, var.names, p
   if ("AbvGrndWood" %in% var.names) {
     forecast[[length(forecast) + 1]] <- udunits2::ud.convert(ens$AbvGrndWood[last],  "kg/m^2", "Mg/ha")
     names(forecast[[length(forecast)]]) <- c("AbvGrndWood")
-   
+    
     # calculate fractions, store in params, will use in write_restart
     wood_total_C    <- ens$AbvGrndWood[last] + ens$fine_root_carbon_content[last] + ens$coarse_root_carbon_content[last]
-    
+    if (wood_total_C==0) wood_total_C <- 0.0001 # Making sure we are not making Nans in case there is no plant living there.    
```

Comment:
@ashiklom This was a @para2x thing. I just added it because it wasn't merged into any of the SDA code. It's supposed to be a temporary fix because values <= 0 were spitting out Inf or Nan values and crashing the SDA. There is a conversation on Slack between @mdietze, @para2x, and (I think @istfer) on this issue and what to do about it in the long run